.orchestra/config/components/source-highlight.yml

```diff
@@ -0,0 +1,43 @@
+#@ load("@ytt:data", "data")
+#@ load("@ytt:overlay", "overlay")
+#@ load("/lib/create_component.lib.yml", "single_build_component")
+#@ load("/lib/make.lib.yml", "make")
+
+#@ source_url = "https://ftp.gnu.org/gnu/src-highlite/source-highlight-3.1.9.tar.gz"
+
+#@yaml/text-templated-strings
+---
+#@ def source_highlight_args():
+license: COPYING
+configure: |
+  mkdir -p "$BUILD_DIR"
+  extract.sh --into "$BUILD_DIR" (@= source_url @)
+  
+  cd "$BUILD_DIR"
+  
+  sed 's/ throw [(]IOException[)]//g' -i lib/srchilite/fileutil.h lib/srchilite/fileutil.cc
```

Comment:
What?

Reply:
`throw` specifiers are deprecated since C++11 and removed in C++17 https://en.cppreference.com/w/cpp/language/except_spec

So either this needs to be done or an old C++ version needs to be forced.

Arch chooses to remove the throw specifiers so that's what I did https://github.com/archlinux/svntogit-packages/blob/packages/source-highlight/trunk/source-highlight-gcc11.patch